app/terramagma/_layout.tsx

```
@@ -0,0 +1,51 @@
+import { Header } from '~/components/layout/header';
+import { Outlet } from 'react-router';
+import { ThemeProvider, useTheme } from '~/components/theme';
+import type { Route } from '../../.react-router/types/app/+types/root';
+import * as React from 'react';
+
+export function meta({}: Route.MetaArgs) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)